### DISCUSSION TITLE
Does changing template variables mean I should set reapply_overlays to true?

### DISCUSSION THREAD

**Discussion by ChristianStrawn** [meisnate12/Plex-Meta-Manager#1489] [Q&A] [answered]
If I change anything in the overlay_path for a library does PMM know it needs to update the poster or do I have to force it with reapply_overlays = true?

I'm assuming it only re-applies changes when the data changes like the number in the ratings change but not when I change the config file?

**Comment by JohnFawkes** [comment, marked as answer]
No you should not. reapply_overlays is almost never needed. It basically means "PMM reapply overlays regardless of if an item has changed or not". PMM will always reapply overlays if a change is detected and won't reapply if no change is needed.
